src/shared/dlt_common.c

```diff
@@ -1793,6 +1795,22 @@ void dlt_log_set_filename(const char *filename)
     logging_filename[NAME_MAX] = 0;
 }
 
+void dlt_log_set_filemode(int filemode)
+{
+    if ((filemode < 0) || (filemode > 1)) {
+        dlt_log(LOG_WARNING, "Wrong parameter: filemode is invalid\n");
+        return;
```

Comment:
well, if I am not wrong, when filemode is invalid, it could cause the behaviour of fopen (line 1846) is undefined.
https://pubs.opengroup.org/onlinepubs/9699919799/functions/fopen.html 

> "The mode argument points to a string. If the string is one of the following, the file shall be opened in the indicated mode. Otherwise, the behavior is undefined."

Reply:
I fixed that, thank you